lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -137,12 +193,12 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
                       height: 120,
                       fit: BoxFit.fill))
                     .text(AppLocalizations.of(context).common_error_occured_message)
-                    .build() : _buildMySpaceListView(context, mySpaceState.documentList)),
+                    .build() : _buildMySpaceListView(context, mySpaceState.documentList.map((element) => element.element).toList())),
```

Comment:
I prefers use `mySpaceState.documentList` `List<SelectableElement<Document>>` instead of convert to `List<Document>`. 

We can use immediately `SelectMode` instead of searching every item build by `isDocumentSelected ()`. It takes a lot of computation although it's unnecessary :(

Reply:
Yes it's not necessary to map the list you're right 

> We can use immediately SelectMode instead of searching every item build by isDocumentSelected ()

Couldn't get ur point can you be more specific please ?

Reply:
Nvm I get it, nice comment

Reply:
u ok with that ? 5e8bc8c76416792688dba828109994cf46e4a99f